internal/bminventory/inventory.go

```diff
@@ -6751,3 +6782,16 @@ func getDefaultHighAvailabilityAndMasterCountParams(highAvailabilityMode *string
 	// both are set
 	return highAvailabilityMode, controlPlaneCount
 }
+
+func extractMirroredRegistriesFromConfig(log logrus.FieldLogger, mirrorRegistryConfig *common.MirrorRegistryConfiguration) []string {
+	var ignorableRegistries []string
+	mirroredRegistries, err := mirrorregistries.ExtractLocationMirrorDataFromRegistriesFromToml(mirrorRegistryConfig.RegistriesConf)
+	if err != nil {
+		log.WithError(err).Warnf("failed to extract location from saved cluster mirror registry configuration while updating cluster params")
```

Comment:
If we fail to extract the mirror registries because of an error, should we swallow this error?